avocado/core/jobdata.py

```diff
@@ -12,127 +12,152 @@
 # Copyright: Red Hat Inc. 2016
 # Author: Amador Pahim <apahim@redhat.com>
 
+"""
+Record/retrieve job information
+"""
+
 import ast
 import glob
-import json
 import os
 import pickle
 import sys
 
-from . import exit_codes
-from .test import ReplaySkipTest
 
+from . import exit_codes
 from .settings import settings
-from ..utils import path
+from ..utils.path import init_dir
 
-"""
-Record/retrieve job information for job replay
-"""
+
+JOB_DATA_DIR = 'jobdata'
+JOB_DATA_FALLBACK_DIR = 'replay'
+CONFIG_FILENAME = 'config'
+URLS_FILENAME = 'urls'
+MUX_FILENAME = 'multiplex'
+PWD_FILENAME = 'pwd'
+ARGS_FILENAME = 'args'
+CMDLINE_FILENAME = 'cmdline'
```

Comment:
Oh, @ldoktor I saw the 'make it private' comment only after send the PR. I will include it in a v3.